### PR TITLE
Updated PieChart, ChainHoldings and TokenList - Mobile/Tablet/ Desktop Responsive & more

### PR DESCRIPTION
**Pull request by manzuni**
- [x] No more content jumping when loading or after on the PieChart, TokenList or ChainHoldings
- [x] Created reusable Card component (used for ChainHoldings , TokenList and PieChart)
- [x] Created reusable Loader component & animation - added it the components using the new Card component
- [x] DRY - Created ListCard component and used it by both TokenList or ChainHoldings
- [x] Continued with making existing components responsive
- [x] Moved Last Updated badge to the top, below header - makes sense since it speaks about the entire page data
- [x]  Also, some desktop view update for the first 3 components. Used available space with more consideration, normalized spaces between components etc.

Next PR, PR3 will most likely deal with the the last major component, HistoricalData, so users can properly read the data no matter the screen size.

image
image
image
image

### CODE REVIEW

File: components/library/Badge.jsx [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import styles from './Badge.module.scss'
```
Comment (manzuni) [inline components/library/Badge.jsx:1]: Reusable Badge component, right now used by the Last Updated text, can be used later on for more

File: components/chainHoldings.js [hunk 2]
```diff
@@ -1,43 +1,24 @@
...
 
   useEffect(() => {
     fetch('/api/data?file=sums.csv&type=summary')
-      .then(response => response.json())
-      .then(data => setData(data))
-      .catch(error => console.error('Error:', error));
-  }, []);
+      .then((response) => response.json())
+      .then((data) => {
+        const sortedData = Object.entries(data)
```
Comment (manzuni) [inline components/chainHoldings.js:13]: Doing the sorting and whatnot only once, on load, when data is pulled is more efficient than doing it on render.

**Comment by Defi-Moses** [comment]
Lets fix the styling in the historical holdings In the first picture (Im not sure which commit broke this). and then fix formatting so that "SYN-ETH Sushi LP" text is cutoff so that the column row formatting doesnt break on desktop. The formatting for historical holdings on mobile completely breaks. Assuming that you dont touch that in this PR?

Other than that looks good and once those two changes are done can merge.
Screenshot 2023-11
<img width=
-01 at 8 35 54 AM" src="https://github.com/Defi-Moses/synapse-treasury-frontend/assets/103143573/32e8738f-dc3e-46e6-90a4-50a350cbb473">

**Comment by manzuni** [comment]
> Lets fix the styling in the historical holdings In the first picture (Im not sure which commit broke this). and then fix formatting so that "SYN-ETH Sushi LP" text is cutoff so that the column row formatting doesnt break on desktop. The formatting for historical holdings on mobile completely breaks. Assuming that you dont touch that in this PR?
> 
> Other than that looks good and once those two changes are done can merge.
> 
> Screenshot 2023-11-01 at 8 35 54 AM Screenshot 2023-11-01 at 8 36 50 AM

Sure, i will push a commit for those and ping you once done.

**Comment by manzuni** [comment]
@Defi-Moses Fixes applied in the last 2 commits:

- [x] Had to touch Historical Data, made it fully responsive and easier to read, this is what it looks like now:
image
image
image
image

- [x] Added stateful Chevron, so it changes from up to down based on isExpanded
- [x] Normalized token/lp text after pulling it in, so we no longer see SushiLp or nothing similar

**Pull request merged**